assets/scss/components/_section.scss

```diff
@@ -11,25 +11,25 @@ $sectionSpaceMultiple: 2;
 
 @include responsiveVarContext--base() {
 	.section {
-		padding: $base*$sectionSpaceMultiple $base 0;
-	}
-	.section--padded {
-		padding-bottom: $base*$sectionSpaceMultiple - $base;
-		&:before{
-			left: $base;
-			right: $base;
+		border-bottom: 1px solid $C_border;
+		margin: 0 $base;
+		padding: $base*$sectionSpaceMultiple 0 $base*$sectionSpaceMultiple - $base 0;
+		&:last-child:not(.section--notAlwaysLast) {
+			border-bottom-width: 0;
+			padding: $base*$sectionSpaceMultiple 0 $base*$sectionSpaceMultiple - $base 0;
 		}
 	}
+	.section--noBorder {
+		border-bottom-width: 0;
+		padding: $base*$sectionSpaceMultiple 0 0 0;
+	}
 }
 
-.section--padded {
-	position: relative;
-	border-top: 0;
-	&:before {
-		content: "";
-		background: $C_border;
-		height: 1px;
-		bottom: 0;
-		position: absolute;
+@include _bpModifier(section, canBeLastChild) {
```

Comment:
We've actually talked about these styles - there's a lot of simplification I need to include.